numba/tests/test_withlifting.py

```diff
@@ -795,6 +798,85 @@ def f():
 
         self.assertEqual(f(), 1 + 3)
 
+    def test_objmode_gv_variable(self):
```

Comment:
might be good to check that the dtypes of the return values are as expected throughout these new tests?

Reply:
i must have accidentally deleted somes lines. The function is never exercised.